engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -1023,9 +1061,31 @@ struct divine_toll_t : public paladin_spell_t
 
 };
 
+struct hallowed_discernment_tick_t : public paladin_spell_t
+{
+  double aoe_multiplier;
+  // This should be using 340203 but I don't have its spell data.
+  hallowed_discernment_tick_t( paladin_t* p ) :
+    paladin_spell_t( "hallowed_discernment", p, p -> find_spell( 317221 ) )
+    {
+      base_multiplier *= p -> conduit.hallowed_discernment.percent();
+      background = true;
+      dual = true;
```

Comment:
Is this necessary? how does this effect show on logs?
If it's an additional AH tick it should probably call an AH tick, if not then its execution count shouldn't be hidden by dual=true